tests/test_polynomial_generic.py

```diff
@@ -38,6 +55,32 @@ def test_reduce_coefficients(self):
             f = self.R(coeffs).reduce_coefficients()
             self.assertTrue(all([c < self.R.q for c in f.coeffs]))
 
+    def test_equality(self):
+        for _ in range(100):
+            f1 = self.R.random_element()
+            f2 = -f1
+            self.assertEqual(f1, f1)
```

Comment:
looks like this one was missed...

Reply:
This one looks like it's doing what I intended

Reply:
ah, ok